WindDirectionControl/WindDirectionControl.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Agate styled WindDirectionControl component and behavior.
+ *
+ * @example
+ * <WindDirectionControl />
+ *
+ * @module agate/WindDirectionControl
+ * @exports WindDirectionControl
+ * @exports WindDirectionControlBase
+ * @exports WindDirectionControlDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import ArcPicker from '../ArcPicker';
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+
+import css from './WindDirectionControl.module.less';
+
+/**
+ * An Agate component for displaying Wind Direction Control {@link agate/WindDirectionControl}.
+ *
+ * @class WindDirectionControl
+ * @memberof agate/WindDirectionControl
+ * @ui
+ * @public
+ */
+const WindDirectionControlBase = kind({
+	name: 'WindDirectionControlBase',
+
+	propTypes: /** @lends agate/WindDirectionControl.WindDirectionControlBase.prototype */ {
+		/**
+		 * Called when value is changed.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		onChange: PropTypes.func,
+
+		/**
+		 * State of WindDirectionControl.
+		 *
+		 * @type {('airDown'|'airRight'|'airUp')}
+		 * @default 'airDown'
+		 * @public
+		 */
+		value:  PropTypes.oneOf(['airDown', 'airRight', 'airUp'])
+	},
+
+	defaultProps: {
+		value: 'airDown'
+	},
+
+	styles: {
+		css,
+		className: 'windDirectionControl'
+	},
+
+	computed: {
+		componentIcon: (props) => {
+			const {value} = props;
+
+			switch (value) {
+				case 'airDown':
+					return 'airdown';
+				case 'airRight':
+					return 'airup';
+				case 'airUp':
+					return 'airright';
+			}
+		}
+	},
+
+	render: ({componentIcon,  onChange, value, ...rest}) => {
+		const children = ['airDown', 'airRight', 'airUp'];
```

Comment:
Let define this out of the kind.
I think that we don't have to define it when calling render().